src/CaptureClient/CaptureEventProcessor.cpp

```diff
@@ -550,12 +552,32 @@ void CaptureEventProcessorForListener::ProcessThreadNamesSnapshot(
   }
 }
 
+ThreadStateSliceInfo::WakeupReason WakeupReasonConversion(
```

Comment:
- This function should be marked `[[nodiscard]]` and `static`
- Also it'd recommend to give it in imperative name. Something like `toWakeupReason`

Reply:
Shouldn't be toWakeUpReason with Uppercase in up?